make/test-e2e.mk

```diff
@@ -72,23 +72,32 @@ endif
 
 .PHONY: e2e-setup-istio
 e2e-setup-istio: | kind-cluster install $(NEEDS_KUBECTL) $(bin_dir)/scratch/istioctl-$(ISTIO_VERSION)
-	$(bin_dir)/scratch/istioctl-$(ISTIO_VERSION) install -y -f ./make/config/istio/istio-config-$(ISTIO_VERSION).yaml
+	$(bin_dir)/scratch/istioctl-$(ISTIO_VERSION) install -y -f ./make/config/istio/istio-config-$(ISTIO_VERSION).yaml $(ISTIO_INSTALL_OPIONS)
```

Comment:
suggestion: There's a typo in `ISTIO_INSTALL_OPIONS` - should be `ISTIO_INSTALL_OPTIONS`.